assets/variant-list.css

```diff
@@ -256,6 +257,17 @@ variant-list-remove-button .icon-remove {
   display: block;
 }
 
+.variant-list-error {
+  margin-top: 1rem;
+  display: flex;
+}
+
+.variant-list-error svg {
+  flex-shrink: 0;
+  width: 1.2rem;
+  margin-right: 0.7rem;
+}
+
 @media screen and (min-width: 750px) {
   .variant-item .loading-overlay {
     padding-top: 2.5rem;
```

Comment:
When there is the generic error, it still lets me add items to cart. Or at least it looks like it (qty isnt reset like the in line error)

Is that expected? https://screenshot.click/30-53-v9bx4-g7bt7.mp4

Reply:
Updated - https://github.com/Shopify/dawn/pull/2754/commits/a67043158380dacbde9b74958ff61f7f29fd51c3